src/providers/Auth/Auth.provider.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useContext } from "react";
+
+export const AuthContext = React.createContext();
+
+// eslint-disable-next-line react/prop-types
+export const AuthProvider = ({ children }) => {
+  const [userData, setUserData] = useState(null);
```

Comment:
To avoid resetting the "authed" variable we can use a resource like LocalStorage, that way if we reload the page. The user wont be redirected to the LoginForm if they are already logged in. Could be something like:
```
let currentAuth = JSON.parse(localStorage.getItem("LocalAuth"));
---
  const [userData, setUserData] = useState(currentAuth ? currentAuth : null);
  const [authed, setAuthed] = useState(currentAuth ? true : false);
```